### PR TITLE
[helm][drbd-adapter]: add drbd-adapter helm templates

### PR DESCRIPTION
**Pull request by SSmallMonster**
#### What this PR does / why we need it:
add drbd-adapter helm templates

#### Does this PR introduce a user-facing change?

```release-note

```

### CODE REVIEW

File: helm/drbd-adapter/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 0.1.0
+
+# This is the version number of the application being deployed. This version number should be
+# incremented each time you make changes to the application. Versions are not expected to
+# follow Semantic Versioning. They should reflect the version the application is using.
+# It is recommended to use it with quotes.
+appVersion: "1.16.0"
```
Comment (yankay) [inline helm/drbd-adapter/Chart.yaml:24]: appVersion and version are same in most cases.

File: helm/drbd-adapter/templates/drbd-adapter.yaml [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+    matchLabels:
+      name: drbd9-adapter
+  template:
+    metadata:
+      labels:
+        name: drbd9-adapter
+    spec:
+      initContainers:
+      - name: shipper
+        image: daocloud.io/daocloud/drbd9-shipper:v9.1.8
```
Comment (yankay) [inline helm/drbd-adapter/templates/drbd-adapter.yaml:18]: opensource project shoud not use image from daocloud.io

**Comment by alexzhc** [comment]
will switch to `ghcr.io`

File: helm/drbd-adapter/Chart.yaml (hunk 1 above)
Comment (SSmallMonster) [reply helm/drbd-adapter/Chart.yaml:24]: fixed

**Pull request merged**